### PR TITLE
Adding to electrical engineering

### PR DESCRIPTION
**Pull request by xcarcelle**

### CODE REVIEW

**Comment by xcarcelle** [comment]
Hi there,
thanks for your great idea of putting this wordlist together which is really useful.  Do my PR fits your rules ?
Best.

File: ReadMe.md [hunk 1]
```diff
@@ -126,6 +126,18 @@ Line | 〜行 | ぎょう | Eg. １３行 (line 13)
...
+Item | 項目 | こうもく |
+Details | 詳細 | しょうさい |
+Requirements | 要求 | ようきゅう |
+Enabled | 有効 | ゆうこう |
+Disabled | 無効 | むこう |
+Factory reset | 初期化	| しょきか |
+Conversion | 変換 | へんかん |
+To control | 制御 (する) | せいぎょ |
+To apply | 適用 (する) | てきよう |
+Static | 静的 | せいてき |
```
Comment (ronkorving) [inline ReadMe.md]: Already mentioned on line 164

File: ReadMe.md [hunk 2]
```diff
@@ -126,6 +126,18 @@ Line | 〜行 | ぎょう | Eg. １３行 (line 13)
...
+Fixed | 固定 | こてい |
+To refer to | 参照 (する) | さんしょう |
```
Comment (ronkorving) [inline ReadMe.md]: Could you move this to line 157/158 (between "Return value" and "Call by value")? That way it's nice and close to "Call by reference" which also mentions 参照

File: ReadMe.md [hunk 3]
```diff
@@ -262,13 +274,16 @@ Capacitor | コンデンサ | | Alternatively: キャパシタ
...
+Communications | 通信 | つうしん |
 Conduction | 導電 | どうでん |
 Conductivity | 導電性 | どうでんせい |
 Connector | コネクタ | |
 Current | 電流 | でんりゅう |
 Datasheet | データシート | |
 Diode | ダイオード | |
 Drawing | 図面 | ずめん |
+Electrolytic	| 電解 | だんかい |
+Frequency	| 周波数 | しゅうはすう|
```
Comment (ronkorving) [inline ReadMe.md]: Sorry to be pedantic, but could you add a space before the closing `|`, and remove the extra whitespace in the first column?

**Comment by ronkorving** [comment]
Hi @xcarcelle! Yes, this fits great!
Some small comments I hope you could address, and then I'll happily hit that big green merge button.

**Comment by xcarcelle** [comment]
@ronkorving : thanks for your comments and very pleased to comply with the comments to make the markdown nicer. I will push more words.

**Comment by ronkorving** [comment]
Thanks!

**Pull request merged**